### PR TITLE
Stories/performance measurement pdfs

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/pdf_generator.rb [hunk 1]
```diff
@@ -17,13 +16,13 @@ def perform(html:, file_name: 'output')
...
+  def render_pdf(html, options: {})
     grover_options = {
       display_url: root_url,
-      displayHeaderFooter: true,
-      headerTemplate: '<h2>Header</h2>',
-      footerTemplate: '<h6 class="text-center">Footer</h6>',
-      timeout: 50_000,
+      display_header_footer: false,
+      header_template: '<h2>Header</h2>',
+      footer_template: '<h6 class="text-center">Footer</h6>',
```
Comment (awisema) [inline app/models/pdf_generator.rb]: Are these just documentation? Maybe comment them out as is done for `margin`?

File: drivers/performance_measurement/app/models/performance_measurement/document_exports/report_export.rb [hunk 2]
```diff
@@ -25,10 +25,20 @@ def authorized?
 
     def perform
       with_status_progression do
-        template_file = 'performance_management/warehouse_reports/report/index_pdf'
+        template_file = 'performance_measurement/warehouse_reports/reports/show_pdf'
+        # https://stackoverflow.com/questions/55865582/set-dynamic-header-and-footer-data-on-pdf-generation-from-rails-grover-gem
         PdfGenerator.new.perform(
-          html: view.render(file: template_file, layout: 'layouts/performance_management'),
+          html: view.render(file: template_file, layout: 'layouts/performance_report'),
```
Comment (awisema) [inline drivers/performance_measurement/app/models/performance_measurement/document_exports/report_export.rb:31]: Should the layout name match the template name `performance_measurement`?

**Review by awisema — CHANGES_REQUESTED** [review]

File: drivers/performance_measurement/app/models/performance_measurement/document_exports/report_export.rb (hunk 2 above)
Comment (eanders) [reply drivers/performance_measurement/app/models/performance_measurement/document_exports/report_export.rb:31]: yes!  I've mis-typed that dozens of times.  Thanks

Comment (eanders) [reply drivers/performance_measurement/app/models/performance_measurement/document_exports/report_export.rb:31]: Oh, actually, I miss-read the diff.  That's correct as is.  It's re-using the layout from the other performance reports.

**Pull request merged**